### PR TITLE
Add procedure howto restore elemental CRD resources

### PR DESCRIPTION
**Pull request by mjura**
Fixes https://github.com/rancher/elemental/issues/447

Signed-off-by: Michal Jura <mjura@suse.com>

### CODE REVIEW

**Review by Itxaka — APPROVED** [review]

File: docs/restore.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Restore
+
+Follow this guide to restore from backup elemental nodes configuration together with Rancher.
```
Comment (nunix) [inline docs/restore.md:3]:
```suggestion
Follow this guide to restore an elemental node configuration from a backup with Rancher.
```

File: docs/restore.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Prepare rancher-backup operator and backup files for restoring
+
+Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher) and make sure that rancher-bakup operator is installed and has access to backup files.
```
Comment (nunix) [inline docs/restore.md:7]:
```suggestion
Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher) and make sure that `rancher-bakup operator` is installed and has access to backup files.
```

File: docs/restore.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Restore elemental with rancher-backup operator
```
Comment (nunix) [inline docs/restore.md:10]:
```suggestion
## Restore the elemental node configuration with rancher-backup operator
```

File: docs/restore.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Restore backup from tarball by creating Restore object
```
Comment (nunix) [inline docs/restore.md:12]:
```suggestion
Create a `restore object` to restore the backup tarball:
```

**Review by nunix — COMMENTED** [review]
Changed some phrases to active voice.

File: docs/restore.md (hunk 1 above)
Comment (mjura) [reply docs/restore.md]: fixed

File: docs/restore.md (hunk 2 above)
Comment (mjura) [reply docs/restore.md]: fixed

File: docs/restore.md (hunk 3 above)
Comment (mjura) [reply docs/restore.md]: fixed

File: docs/restore.md (hunk 4 above)
Comment (mjura) [reply docs/restore.md]: fixed

**Review by nunix — APPROVED** [review]

**Pull request merged**